### DISCUSSION TITLE
What is the equivalent of `ReactTooltip.rebuild()` from v4 to v5?

### DISCUSSION THREAD

**Discussion by andrew-tacklit** [ReactTooltip/react-tooltip#892] [Q&A] [answered]
Hi, I'm trying to migrate from v4 to v5, however I couldn't find information regarding the `.rebuild()` function in v5 documentation.

May I know the equivalent of `ReactTooltip.rebuild()` from v4 to v5? Thank you 🙏

**Comment by gabrieljablonski** [comment, marked as answer]
Hi Andrew.

`ReactTooltip.rebuild()` existed as a workaround given some limitations the v4 design had.

v5 was a complete redesign of the component, so `ReactTooltip.rebuild()` isn't needed anymore.

In case you're facing any problems, feel free to open an issue.
